webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -192,6 +241,22 @@ export const ErrataTab = () => {
                     </Td>
                     <Td><ErrataType {...erratum} /></Td>
                     <Td><ErrataSeverity {...erratum} /></Td>
+                    <Td>
+                      { isInstallable ?
+                        <span><CheckIcon /> {__('Yes')}</span> :
+                        <span>
+                          <Tooltip
+                            content={
+                                         __("This Erratum is not installable as it is not in this Host's Content View and Lifecycle Environment.")
```

Comment:
```suggestion
                                         __("This erratum is not installable because it is not in this host's content view and lifecycle environment.")
```